src/main/java/seedu/address/logic/commands/ListProjectsCommand.java

```diff
@@ -0,0 +1,25 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
```

Comment:
Should the predicate being imported be PROJECTS instead?

Reply:
yup good spot.